src/prepack-cli.js

```diff
@@ -541,7 +668,11 @@ fi
     }
   }
 
-  if (!success) process.exit(1);
+  // If there is a repro going on, don't exit.
+  // The repro involves an async directory zip, so exiting here will cause the repro
+  // to not complete. Instead, all calls to repro include a flag to indicate
+  // whether or not it should process.exit() upon completion.
+  if (!success && reproMode === undefined) process.exit(1);
```

Comment:
Instead of calling process.exit() coordinated, let's do zipping as the last step of Prepack and call process.exit() there.